docker/README.md

```diff
@@ -1,10 +1,35 @@
-[![Build Status](https://travis-ci.org/CellProfiler/docker.svg?branch=master)](https://travis-ci.org/CellProfiler/docker)
-
 Testing docker locally:
 
     make
+    make clean
     make test
 
+Note: `make test` will work only for CellProfiler 3.3.0 and higher because the example pipeline in the test is not backward compatible.
+
+This is configured to build a docker image for `3.0.0`. To build a different version, edit these lines:
+
+Makefile:
+
+    VERSION := 3.0.0
+
+Dockerfile:
+
+    ARG version=3.0.0
+    
+You can do this using `sed` e.g.
+    
+    $ export VERSION=2.2.1
+    $ sed -i s,"ARG version=3.0.0","ARG version=${VERSION}",g Dockerfile
```

Comment:
I get the following error when I run this command:

```
$ sed -i s,"ARG version=3.0.0","ARG version=${VERSION}",g Dockerfile
sed: 1: "Dockerfile": extra characters at the end of D command
```

Reply:
I'm wondering whether this is because of gsed vs sed on OSX. Can you check? If that's the issue. I'll update docs with instructions
https://stackoverflow.com/questions/30003570/how-to-use-gnu-sed-on-mac-os-x

Reply:
Bingo!

I ran `brew install gnu-sed` and added gnu bin to my path (`PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"`) to use the command as `sed` instead of `gsed`.

Reply:
Great. I've updated the docs